docs/content/SplitPageLayout.mdx

```diff
@@ -0,0 +1,313 @@
+---
+title: SplitPageLayout
+componentId: split_page_layout
+status: Draft
+description: Provides structure for a split page layout, including independent scrolling for the pane and content regions. Useful for responsive list/detail patterns, when an item in the pane updates the page content on selection.
+---
+
+<Note variant="warning">Not implemented yet</Note>
+
+## Examples
+
+### Default
+
+```jsx
+<SplitPageLayout>
+  <SplitPageLayout.Header>
+    <Placeholder label="Header" height={64} />
+  </SplitPageLayout.Header>
+  <SplitPageLayout.Pane>
+    <Placeholder label="Pane" height={120} />
+  </SplitPageLayout.Pane>
+  <SplitPageLayout.Content>
+    <Placeholder label="Content" height={240} />
+  </SplitPageLayout.Content>
+  <SplitPageLayout.Footer>
+    <Placeholder label="Footer" height={64} />
+  </SplitPageLayout.Footer>
+</SplitPageLayout>
+```
+
+### With pane hidden on narrow viewports
+
+```jsx
+<SplitPageLayout>
+  <SplitPageLayout.Header>
+    <Placeholder label="Header" height={64} />
+  </SplitPageLayout.Header>
+  <SplitPageLayout.Pane visible={{narrow: false}}>
+    <Placeholder label="Pane" height={120} />
+  </SplitPageLayout.Pane>
+  <SplitPageLayout.Content>
+    <Placeholder label="Content" height={240} />
+  </SplitPageLayout.Content>
+  <SplitPageLayout.Footer>
+    <Placeholder label="Footer" height={64} />
+  </SplitPageLayout.Footer>
+</SplitPageLayout>
+```
+
+### With content hidden on narrow viewports
+
+```jsx
+<SplitPageLayout>
+  <SplitPageLayout.Header>
+    <Placeholder label="Header" height={64} />
+  </SplitPageLayout.Header>
+  <SplitPageLayout.Pane>
+    <Placeholder label="Pane" height={120} />
+  </SplitPageLayout.Pane>
+  <SplitPageLayout.Content visible={{narrow: false}}>
+    <Placeholder label="Content" height={240} />
+  </SplitPageLayout.Content>
+  <SplitPageLayout.Footer>
+    <Placeholder label="Footer" height={64} />
+  </SplitPageLayout.Footer>
+</SplitPageLayout>
+```
+
+### Without dividers
+
+```jsx
+<SplitPageLayout>
+  <SplitPageLayout.Header divider="none">
+    <Placeholder label="Header" height={64} />
+  </SplitPageLayout.Header>
+  <SplitPageLayout.Pane divider="none">
+    <Placeholder label="Pane" height={120} />
+  </SplitPageLayout.Pane>
+  <SplitPageLayout.Content>
+    <Placeholder label="Content" height={240} />
+  </SplitPageLayout.Content>
+  <SplitPageLayout.Footer divider="none">
+    <Placeholder label="Footer" height={64} />
+  </SplitPageLayout.Footer>
+</SplitPageLayout>
```

Comment:
for later (and for PageLayout): can we change this example to show how to customise dividers (filled, line, none) instead of simply turning them off.

Reply:
> the default for divider is line in SplitPageLayout, but none in PageLayout, is this am intentional design inconsistency?

I believe this is intentional. @vdepizzol feel free to correct me if I'm wrong

> for later: can we change this example to show how to customise dividers (filled, line, none) instead of simply turning them off.

👍

Reply:
It's intentional.